packages/scss/src/components/table/mods.scss

```diff
@@ -153,33 +153,35 @@
 }
 
 @mixin draggable {
-	.table-body-row-cell-handler {
-		top: 0;
-		bottom: 0;
-		left: 0;
-		width: 2rem;
-		position: absolute;
-		cursor: move;
-
-		&:focus-visible,
-		&:hover {
-			&::after {
-				color: var(--palettes-grey-500);
+	@at-root {
+		.table-body-row-cell-handler {
+			top: 0;
+			bottom: 0;
+			left: 0;
+			width: 2rem;
+			position: absolute;
+			cursor: move;
+
+			&:focus-visible,
+			&:hover {
+				&::after {
+					color: var(--palettes-grey-500);
+				}
 			}
-		}
 
-		&::after {
-			@include icon.makeIcon('drag');
+			&::after {
+				@include icon.makeIcon('drag');
 
-			color: var(--palettes-grey-400);
-			height: 100%;
-			margin: auto;
-			display: flex;
-			align-items: center;
+				color: var(--palettes-grey-400);
+				height: 100%;
+				margin: auto;
+				display: flex;
+				align-items: center;
+			}
 		}
 	}
 
-	[class~='-row-cell'] {
+	[class$='-row-cell'] {
```

Comment:
`$=` shouldn't be used has it make the rule dependant to the class order in the HTML. See an example here : https://codepen.io/bertrandpodevin/pen/KKQqWON
My rule of thumb is to only use `*=` to avoid those problems

Reply:
Well seen, thank you!